### DISCUSSION TITLE
New Main 2 branch

### DISCUSSION THREAD

**Discussion by ElangBogopa** [CSC207-2022F-UofT/course-project-npcs-ratemyuoft-project#19] [General]
Hey guys, before committing everything to main, I made a separate branch and uploaded it so you guys can review it and then Ill merge it with main branch if we are happy with it.

**Comment by IamMichael23** [comment]
got it

**Comment by myk0laUA** [comment]
Don't forget to ask someone to review your code and approve merging. So that, there would be no conflicts or trouble with merging.
